python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1169,16 +1164,63 @@ async def get_m1m3_bump_test_status(
             timeout=self.fast_timeout
         )
 
-        actuator_index = self.get_m1m3_actuator_index(status.actuatorId)
+        return self._extract_bump_test_status_info(status)
+
+    def _extract_bump_test_status_info(
```

Comment:
This can be greatly simplified when you remember the actuator ID and pass that as an argument, preferably to get_m1m3_bump_test_status. You use get_m1m3_bump_test_status only here and then in tests, always know actuator_id because that's for what you started test. Please add to get_m1m3_bump_test_status this actuator_id argument, translate that to primary and secondary indices, get bump test status and you are done. You don't need to rely on complex max operation from timestamps.